Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -263,6 +263,15 @@ theorem zero_or_succ_or_limit (o : Ordinal) : o = 0 ∨ (∃ a, o = succ a) ∨
     if h : ∃ a, o = succ a then Or.inr (Or.inl h)
     else Or.inr <| Or.inr ⟨o0, fun _a => (succ_lt_of_not_succ h).2⟩
 
+theorem IsLimit.sSup_Iio {o : Ordinal} (h : IsLimit o) : sSup (Iio o) = o := by
```

Comment:
I'd rather not introduce anything about `IsSuccPrelimit` / `IsSuccLimit` in this file until `IsLimit` is deprecated, otherwise it'll just make it annoying having to switch between both APIs.

The `IsLimit` version isn't an iff as `sSup (Iio 0) = 0`.

Reply:
Eh, okay. Can you leave a TODO then? And I meant that the prelimit version is an iff